azure-pipelines/azure-pipeline.microbuild.before.yml

```diff
@@ -1,10 +1 @@
-steps:
-
-- task: MicroBuildSigningPlugin@3
-  inputs:
-    signType: $(SignType)
-    zipSources: false
-  displayName: Install MicroBuild Signing Plugin
-
-- task: MicroBuildSwixPlugin@4
-  displayName: Install MicroBuild Swix Plugin
+steps: []
```

Comment:
Should we just delete this file (and the reference to it)? I guess no harm in leaving it if its needed in the future. Just seems odd having an (essentially) empty file here.